src/Components/AdvancedSearch/Internal/AdvancedSearchResultDetailsList/AdvancedSearchResultDetailsList.tsx

```diff
@@ -71,39 +77,46 @@ const AdvancedSearchResultDetailsList: React.FC<IAdvancedSearchResultDetailsList
         switch (column.key) {
             case 'properties':
                 return (
-                    <IconButton
-                        iconProps={{ iconName: 'EntryView' }}
-                        title={t('advancedSearch.inspectProperty')}
-                        ariaLabel={t('advancedSearch.inspectProperty')}
-                        onClick={(event) => {
-                            event.stopPropagation();
-                        }}
-                        className={'cb-scenes-action-button'}
+                    <PropertyInspectorCalloutButton
+                        twinId={`${currentTwin?.$dtId}`}
```

Comment:
this does not pass the right twinId. It's showing the data as undefined. We should be able to use the `item` passed into this function instead of managing another state variable.
![image](https://user-images.githubusercontent.com/57726991/185501995-a4deaa89-4766-44f1-871a-56750b057bf4.png)